data/categories.txt

```diff
@@ -3527,6 +3527,11 @@ vi:Đá ranh giới
 zh-Hans:界碑
 zh-Hant:界碑
 
+boundary-marker
+en:Boundary Marker
```

Comment:
this actually enables searching for boundary markers

but do we really need it?

Reply:
IMO we should minimize adding this kind of elements to avoid slowing down the search.

Reply:
I aggree, removed that change